### ISSUE TITLE
distance matrice

### ISSUE THREAD

**Issue by TyrandeWhisperwind** [letiantian/kmedoids#9]
what is the tmax varaible for? and why did u set it to 100?
if i have a dataset of 800 points should i put tmax=800?

**Comment by casibus** [comment]
No. tmax is just the maximal number of iterations in which the algorithm can converge. Usually kmedoids converges quickly, but for your specific usecase you can play around with the parameter if you have the feeling it did not converge properly.
